cmake/BLTPrivateMacros.cmake

```diff
@@ -798,29 +832,30 @@ macro(blt_split_source_list_by_language)
             message(FATAL_ERROR "blt_split_source_list_by_language given source file with no extension: ${_file}")
         endif()
 
-        get_filename_component(_name "${_file}" NAME)  
-
-        string(TOLOWER "${_ext}" _ext_lower)
-
-        if("${_ext_lower}" IN_LIST BLT_C_FILE_EXTS)
-            if (DEFINED arg_C_LIST)
```

Comment:
Can you please indent by 4 spaces since that is what we do across the project? (or at least i fix it when i see it)

Reply:
Should be updated in [`f407f0c2`](https://github.com/LLNL/blt/pull/528/commits/f407f0c2be641477f37a9b7d5fb273168ebf6c8a)?